### PR TITLE
Support explicitly passing in local files in options.stylsheets

### PR DESCRIPTION
**Pull request by XhmikosR**
@RyanZim @mikelambert: feel free to push to this branch and merge when done.

### CODE REVIEW

**Comment by mikelambert** [comment]
So is the format `file:/path/to/file`? This seems rather...nonstandard. Shouldn't it be `file:///path/to/file/`, to match the use of `file` schemas as I see them in the browser and other places?

(The unittest didn't seem to do a `file:///` path, afaict)

**Comment by RyanZim** [comment]
@XhmikosR Thoughts on @mikelambert's comment?

**Comment by XhmikosR** [comment]
The unit test might need adaption. Not sure which solution is "better" but I guess there's an official way to reference files.

**Comment by RyanZim** [comment]
@XhmikosR How is this going to work/look on windows?

**Comment by XhmikosR** [comment]
I don't think there will be any difference. I mean, tests are passing.

Any way, I'm not the author of this patch, I just resolved conflicts and stuff.

**Comment by RyanZim** [comment]
> Any way, I'm not the author of this patch, I just resolved conflicts and stuff.

Yeah sorry, forgot about that. I guess I'll leave this to @mikelambert. At the end of the day, IDC as long as it doesn't create regressions.

File: tests/url.js [hunk 1]
```diff
@@ -69,6 +69,29 @@ describe('Compile the CSS of an HTML page passed by URL', () => {
         });
     });
 
+    it('Deals with local options.stylesheets when using URLs', function (done) {
+        const localStylesheetPath = path.join(__dirname, '/input/main.css');
+
+        this.timeout(25000);
+        uncss(['http://giakki.github.io/uncss/'], {
+            stylesheets: [path.join('file:', localStylesheetPath)]
```
Comment (mikelambert) [inline tests/url.js]: I still have my same concern mentioned above, about how we reference local files: https://en.wikipedia.org/wiki/File_URI_scheme

Namely, should be `file://host/path/`, or in the case of local files, should be `file:///path/`.

**Comment by tobek** [comment]
I agree that it should use be `file://`, my bad for using `file:` in my original pull request. As it turns out, the code as it is accepts `file:///path/to/local/file.css` just fine, so updating the docs and tests were all that's needed.

I wasn't sure how to amend this pull request (looks like that's not possible for me since the PR is within this repo?) but I updated `master` on my branch and have submitted another PR #348.

I haven't tested it on Windows since I don't have a Windows machine. I think the principle will work, and that the path would have to be `file://c:/path/to/file` or `file:///c:/path/to/file` (not sure which) but a Windows user should be familiar with that syntax and will know to supply it, and I *think* that `__dirname` in the test will ensure the test passes on Windows.

**Pull request closed without merging**